lib/browser/api/desktop-capturer.ts

```diff
@@ -13,6 +13,13 @@ function isValid (options: Electron.SourcesOptions) {
   return Array.isArray(options?.types);
 }
 
+const systemPickerVideoSource = Object.create(null);
+systemPickerVideoSource.id = 'window:0:0';
+systemPickerVideoSource.name = '';
+Object.freeze(systemPickerVideoSource);
```

Comment:
Could we maybe call out that this is specifically needed for the MacOS system picker implementation? I could see us coming back to this in 6 months and being unsure of why we needed to do this 🙇‍♀️